example_system_card/system_card.yaml

```diff
@@ -75,4 +82,5 @@ models:
 
 assessments:
   - !include assessments/iama.yaml
+  - !include assessments/dpia.yaml
```

Comment:
Maybe also include this one in this PR? Like the real assessment or some kind of mock, I see this to be failing something else

Reply:
We don't have a mock of the DPIA, so I'll take it out for now